src/views/annual-report/2020/l10n.json

```diff
@@ -0,0 +1,379 @@
+    "annualReport.2020.mastheadTitle": "Adapting to a Changing World",
+    "annualReport.2020.foundersMessageTitle": "A Message from Our Founder",
+    "annualReport.2020.foundersMessageP1": "The year 2020 will be remembered as the year when the COVID pandemic swept across the world, causing hardships and disruptions in the lives of everyone -- with the greatest hardships falling inequitably on those already facing challenges in their lives.",
+    "annualReport.2020.foundersMessageP2": "Throughout the pandemic, young people around the world, many isolated in their homes, have come to the Scratch website in greater numbers than ever before, seeing Scratch as a safe space where they can express themselves creatively, learn new skills, and collaborate with one another. We were inspired by so many of the Scratch projects that young people created during 2020, many of them sharing their thoughts and feelings about the pandemic, climate change, racial injustice, and other issues on their minds. Young people were not just learning computational concepts and skills, but also developing their voice and their identities.",
+    "annualReport.2020.foundersMessageP3": "To ensure that Scratch can continue to play this important role in young people’s lives in the years ahead, we’ve been making significant organizational changes at Scratch. At the start of 2020, the Scratch Team moved out of its longtime home at the MIT Media Lab and into the new offices of the Scratch Foundation in downtown Boston. This move will help us to build a sustainable organization capable of supporting Scratch as a global creative coding platform into the future.",
+    "annualReport.2020.foundersMessageP4": "Later in 2020, as part of this organizational transition, we hired Shawna Young to serve as Executive Director of the Scratch Foundation. Shawna comes to the Scratch Foundation with a strong background in education and nonprofit management, and a deep commitment to equity and inclusion. Throughout her career in institutions such as Duke and MIT, Shawna has worked to expand learning experiences for students from diverse communities. That commitment is strongly aligned with Scratch’s mission and values, and it will play an important role in her leadership at Scratch. I encourage you to read Shawna’s message at the end of this annual report.",
+    "annualReport.2020.foundersMessageP5": "Over the past decade, Scratch has had phenomenal success, engaging tens of millions of young people around the world. But we are just beginning. The challenge for the years ahead is to ensure that we can continue to spread and support not just our technology but also our creative, caring, collaborative learning approach, so that young people around the world have equitable opportunities to imagine, create, share, and learn. We look forward to working with all of you to make that happen!",
+    "annualReport.2020.foundersMessageSignature": "Mitch Resnick",
```

Comment:
This string can just go into the source code since we wouldn't be translating this name or any other strings that are just standalone proper nouns (besides country names)